products/drupal.md

```diff
@@ -121,6 +134,6 @@ bugs and security issues that have been reported are fixed and are released duri
 (US time). The final minor release in major release cycle is a long-term support (LTS) release and
 has extended security coverage.
 
-**Drupal 7** will receive security coverage [until January 5th 2025](https://www.drupal.org/about/core/policies/core-release-cycles/schedule#s-drupal-7-and-9-end-of-life-dates)
-and [limited vendor extended support](https://www.drupal.org/psa-2023-06-07). This will be the final
-extension.
+**Drupal 7** will receive security coverage [until January 5th 2025](https://www.drupal.org/about/core/policies/core-release-cycles/schedule#s-drupal-7-and-9-end-of-life-dates) with commercial support available post end-of-life through Drupal's [Certified D7 End of Life Support Partner](https://www.drupal.org/about/drupal-7/d7eol/partners#commercial-support) HeroDevs. For more information about commercial support options for Drupal 7 visit [HeroDevs' Website](https://www.herodevs.com/support/nes-drupal).
+
+This will be the final extension of support for Drupal 7.
```

Comment:
```suggestion
**Drupal 7** will receive security coverage [until January 5th 2025](https://www.drupal.org/about/core/policies/core-release-cycles/schedule#s-drupal-7-and-9-end-of-life-dates) with commercial support available post end-of-life through Drupal's [Certified D7 End of Life Support Partner](https://www.drupal.org/about/drupal-7/d7eol/partners#commercial-support).
```

We aim to maintain a neutral and informative tone on endoflife.date, without promotional intent. Given Drupal is providing a page listing various support options, including Drupal 7 NES from HeroDevs, let's just mention it but not give any details.